rules/linux/command_and_control_tunneling_via_earthworm.toml

```diff
@@ -0,0 +1,44 @@
+[metadata]
+creation_date = "2021/04/12"
+maturity = "production"
+updated_date = "2021/04/12"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the execution of the EarthWorm tunneler. Adversaries may tunnel network communications to and from a victim
+system within a separate protocol to avoid detection and network filtering, or to enable access to otherwise unreachable systems.
+"""
+from = "now-9m"
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Potential Protocol Tunneling via EarthWorm"
+references = [
+    "http://rootkiter.com/EarthWorm/",
+    "https://decoded.avast.io/luigicamastra/apt-group-targeting-governmental-agencies-in-east-asia/"
+]
+risk_score = 47
+rule_id = "9f1c4ca3-44b5-481d-ba42-32dc215a2769"
+severity = "medium"
+tags = ["Elastic", "Host", "Linux", "Threat Detection", "Command and Control"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+process where event.type == "start" and
```

Comment:
```suggestion
process where event.type == "start" and process.name == "ew" and
```

Reply:
process.name can be renamed, will make it easy to bypass.

Reply:
any process can be renamed 🤷